SU2_CFD/src/solvers/CSolver.cpp

```diff
@@ -1764,8 +1765,11 @@ void CSolver::AdaptCFLNumber(CGeometry **geometry,
 
     /* Check if we should decrease or if we can increase, the 20% is to avoid flip-flopping. */
     resetCFL = linRes > 0.99;
-    reduceCFL = linRes > 1.2*linTol;
-    canIncrease = linRes < linTol;
+    
+    /* only change CFL number when larger than starting iteration */
+    reduceCFL = linRes > 1.2*linTol && (config->GetOuterIter() > startingIter);
+
+    canIncrease = (linRes < linTol) && (config->GetOuterIter() > startingIter);
```

Comment:
using outer iter only works for multizone right?

Reply:
Yes, it is, my bad. Let's introduce the more general iteration counter here.